Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Next
 ### Added
 - **Breaking Change** Added a `.requestCustomJSONEncodable` case to `Task`. [#1443](https://github.com/Moya/Moya/pull/1443) by [@evgeny-sureev](https://github.com/evgeny-sureev).
+- Added `failsOnEmptyData` boolean support for the `Decodable` map functions. [#1508](https://github.com/Moya/Moya/pull/1508) by [@jeroenbb94](https://github.com/Jeroenbb94).
```

Comment:
This is a breaking change because we changed the function signatures of the `map` methods

Reply:
Is it? It's only added functionality. It doesn't break any existing functionality 🤔

Reply:
Yep, it doesn't break any functionality but it breaks the API. The map method has a new parameter, which is a change in the function's signature, and therefore it is a different type.

Reply:
Thx, I'll adjust it